### DISCUSSION TITLE
'flutterfire configure' FirebaseCommandException:

### DISCUSSION THREAD

**Discussion by Ericos-404** [firebase/flutterfire#9777] [Q&A]
When I try to run the `flutterfire configure` statement and after I type in the project id for my new Firebase project I get this Error:
FirebaseCommandException: An error occured on the Firebase CLI when attempting to run a command.
COMMAND: firebase projects:create mynotes-camp-code --json 
ERROR: Failed to create project. See firebase-debug.log for more info.

Thanks for your help

**Comment by mustafaergul0** [comment]
Did you solve the problem?
